src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp

```diff
@@ -562,12 +581,17 @@ std::vector<size_t> patternTypesQuant = {
     0, 1
 };
 
+std::vector<bool> fakeQuantize3Exists = {
+    false, true
+};
+
 INSTANTIATE_TEST_SUITE_P(smoke_MHAQuant, MHAQuantTest,
                         ::testing::Combine(
                                 ::testing::ValuesIn(static_shapes_to_test_representation(inputShapesQuant)),
                                 ::testing::ValuesIn(inputPrecisionsQuant),
                                 ::testing::ValuesIn(matMulIn0PrecisionsQuant),
                                 ::testing::ValuesIn(patternTypesQuant),
+                                ::testing::ValuesIn(fakeQuantize3Exists),
```

Comment:
We have tests duplication for case of  `quantPatternType=0` since fakeQuantize3Exists is not actually used in that case.
I would propose either cover new that case via new quant pattern type or separate the instances.

Reply:
done, thanks